config.yaml.dist

```diff
@@ -31,6 +31,16 @@ clusterNetwork:
   serviceDomainName: ""
   # a nodePort range to reserve for services (default: 30000-32767)
   nodePortRange: ""
+  # CNI plugin choise
+  cni:
+    # possible values:
+    # * canal
+    # * weave-net
+    provider: canal
+    # when selected CNI provider support encryption and `encrypted: true` is
```

Comment:
Mention which CNI plugins support encryption, i.e. that it's currently only Weave.